src/YieldDistributor.sol

```diff
@@ -108,12 +109,13 @@ contract YieldDistributor is IYieldDistributor, OwnableUpgradeable {
      * @return uint256 The voting power of the user
      */
     function getCurrentVotingPower(address _account) public view returns (uint256) {
-        return this.getVotingPowerForPeriod(
-            BREAD, lastClaimedBlockNumber - cycleLength, lastClaimedBlockNumber, _account
-        )
-            + this.getVotingPowerForPeriod(
-                BUTTERED_BREAD, lastClaimedBlockNumber - cycleLength, lastClaimedBlockNumber, _account
-            );
+        uint256 lastCycleStart = lastClaimedBlockNumber - cycleLength;
+        uint256 multiplier = this.getTotalMultipliers(_account);
+        multiplier = multiplier == 0 ? 1e18 : multiplier;
```

Comment:
Two issues:
(1) This should be 1 instead of `1e18` otherwise you're going to be increasing the resultant voting power by `e18`.
(2) You should just be able to use to multiplier result directly to multiply the voting power, so this ternary should be part of the `getTotalMultipliers` function.